py3status/cli.py

```diff
@@ -101,6 +101,13 @@ def parse_cli():
         type=int,
         default=60,
         help="default injection cache timeout in seconds (default 60 sec)")
+    parser.add_argument(
+        '-m',
+        '--disable-mouse',
+        action="store_false",
+        dest="enable_mouse",
+        default=True,
+        help="disable all mouse events")
```

Comment:
Maybe just `disable mouse events`? Or @tobes, do you prefer broad `disable button clicks` instead?